src/components/Avatar/Avatar.style.ts

```diff
@@ -0,0 +1,84 @@
+import { styled } from 'styled-components';
+
+export type Shape = 'circle' | 'round' | 'square';
+export type Size = 'XS' | 'S' | 'M' | 'L' | 'XL';
+
+export interface AvatarProps {
+  size?: Size;
+  shape?: Shape;
+  src?: string;
+  isBorder?: boolean;
+  isShadow?: boolean;
+  isEdit?: boolean;
+  onClick?: (event: React.MouseEventHandler<HTMLElement>) => void;
+}
+
+const ShapeToStyle: { [key: string]: string } = {
+  circle: '50%',
+  round: '5px',
+  square: '0px',
+};
+
+const SizeToStyle: { [key: string]: string } = {
+  XS: '26px',
+  S: '36px',
+  M: '46px',
+  L: '56px',
+  XL: '66px',
+};
+
+export const AvatarWrapper = styled.div<{
+  size?: Size;
+  shape?: Shape;
+  src?: string;
+  $isBorder?: boolean;
+  $isShadow?: boolean;
+}>`
```

Comment:
src를 제외하면 Avatar 컴포넌트에서
default props로 값을 무조건 주고 있어서 이렇게 변경해볼 수 있을 것 같아요

옵셔널 체이닝을 제거해주면 아래 코드도 줄여볼 수 있을 것 같습니다 👍
이어서 코멘트 드릴게요!

```suggestion
export const AvatarWrapper = styled.div<{
  size: Size;
  shape: Shape;
  src?: string;
  $isBorder: boolean;
  $isShadow: boolean;
}>`
```

Reply:
👍 생각해보지 못했네요! AvatarProps를 재사용할 필요 없이 interface를 별도로 선언해서 해당 부분 적용해보겠습니다!